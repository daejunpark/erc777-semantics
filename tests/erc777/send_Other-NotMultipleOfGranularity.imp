test.preamble();
test.setGranularity(10)

// Testing send(To, Value, "") when To != caller and Value > 0.

test.send(4, 23, "");
