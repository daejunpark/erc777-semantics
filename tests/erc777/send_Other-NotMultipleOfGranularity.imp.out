granularity: 10
7 is sending 23 to 4 (data:)
REVERT
Balance of 7 is 70
Balance of 4 is 40
